src/services/experimental-features.js

```diff
@@ -0,0 +1,10 @@
+export class ExperimentalFeatures {
+  constructor(storage) {
+    this.features = storage.getItem('dopplerExpermiental')
+      ? JSON.parse(storage.getItem('dopplerExpermiental'))
+      : null;
+  }
+  getFeature(featureName) {
+    return this.features && !!this.features[featureName] ? this.features[featureName] : null;
```

Comment:
What do you think about this:

```suggestion
    return this.features && this.features[featureName] || null;
```